Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "LibPQ"
 uuid = "194296ae-ab2e-5f79-8cd4-7183a0a5a0d1"
 license = "MIT"
-version = "1.0.7"
+version = "1.1.0"
```

Comment:
In the future, please don't bump the version in a PR.

Reply:
I assume you mean don't bump major / minor versions in a PR?

Reply:
No, I mean don't change the version in a PR at all

Reply:
I'm curious why? I thought that bumping the version number here is a typical part of the workflow?

Reply:
Some people do that and I disagree with it. 

It causes MR conflicts, can introduce false Project.toml information (if the version doesn't get registered or there's another version bump before the next version), and can make certain assertions about the breakingness of the release as a whole (not just the current MR) that may not be true.